### DISCUSSION TITLE
Score Results Screen Displays Standardised Initially Regardless of Score Display Mode

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16934] [Q&A] [answered]
Version 2022.217.0
This appears to be unintended behavior. The score display mode was not changed throughout the map, but the file size limitations had me cut the length down.

https://user-images.githubusercontent.com/74262407/154869639-d61e2b84-a33b-4046-9079-b4f4ca3561a7.mp4

Not to mention the fact that osu!taiko Classic Score Display doesn't reflect the behavior of stable, but I suppose that can be added to #14794.

**Comment by peppy** [comment]
This is really weird.. I can't reproduce locally, regardless of whether playing with online scores present, regardless of osu! or taiko ruleset.

Could this be some kind of race condition due to asynchronous calculation?

**Comment by smoogipoo** [comment, marked as answer]
Should be the same cause as https://github.com/ppy/osu/issues/17549. Should be fixed via https://github.com/ppy/osu/pull/17550
